src/components/ground-nav/ground-nav.twig

```diff
@@ -1,92 +1,172 @@
+    <div class="c-ground-nav__features o-container o-container--pad-inline">
+      <div class="c-ground-nav__features-layout o-container__content">
+        <div class="c-ground-nav__features-inner t-dark o-container__fill">
+          <div class="c-ground-nav__feature o-container__pad
+                      {% if features > 1 %}t-alternate{% endif %}">
+            {{ feature_one }}
+          </div>
+          {% if features > 1 %}
+            <div class="c-ground-nav__feature o-container__pad">
+              {{ feature_two }}
+            </div>
+          {% endif %}
         </div>
       </div>
-    {% endwith %}
+    </div>
   {% endif %}
 
-  <div class="c-ground-nav__content o-container o-container--pad t-alternate">
+  {#
+    Ground Nav Content:
+    1. Usage of the `address` element
+      @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address
+    2. Format phone numbers for click-to-call
+      @see https://web.dev/click-to-call/
+    3. Prefer `aria-labelledby` to `aria-label` for translation
+      @see https://heydonworks.com/article/aria-label-is-a-xenophobe/
+    4. We want different text for screenreaders than sighted users.
+      For sighted users "Explore" is a better label, but for screenreaders,
+      who may be navigating by roles or headings, indicating that this menu
+      contains the main menu contents plus some links is more meaningful.
+    5. Add `role="list"` to list elements to retain screenreader semantics
+      @see https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html
+    6. Add `focusable="false"` to SVGs to retain screenreader semantics
+      @see https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html#svgs-that-are-decorative
+  #}
+
+  <div class="c-ground-nav__content o-container o-container--pad">
     <div class="c-ground-nav__content-inner o-container__content">
-      <h2 class="u-hidden-visually">
-        Contact info
-      </h2>
-      {# https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address #}
+
       {% with organization|default({}) %}
-        <address class="c-ground-nav__organization">
-          <a href="{{ url }}" class="c-ground-nav__organization-name">
-            {{ name }}
-          </a><br>
-          {% with address|default({}) %}
-            {{ street_address }}<br>
-            {{ address_locality }}, {{ address_region }} {{ postal_code }} {{ address_country }}<br>
-          {% endwith %}
-          <a href="mailto:{{ email }}">{{ email }}</a><br>
-          {# https://developers.google.com/web/fundamentals/native-hardware/click-to-call/ #}
-          <a href="tel:{{ telephone|replace({
-            '.': '-',
-            '(': '-',
-            ')': '-',
-            ' ': ''
-          }) }}">{{ telephone }}</a>
-        </address>
+        <div class="c-ground-nav__address">
+          <h2 class="u-hidden-visually">
+            Contact info
+          </h2>
+          <address>{# 1 #}
```

Comment:
I was reviewing with VoiceOver and I'm wondering if we can restructure this just a tad.

Right now, I hear "Contact info" but I don't see any heading. For users who may have low-vision (not blind), it might be confusing because the screen reader focus ring gets lost. 

How about the following as an alternative?

```html
<h2 class="u-hidden-visually" aria-hidden="true" id="contact-info-heading">
  Contact info
</h2>
<address aria-labelledby="contact-info-heading">
```
- Adds `aria-hidden` to the H2
- Adds an ID to the H2
- Adds `aria-labelledby` to the `address` element

With this UX, I hear "Contact info, group", then the next screen reader navigation takes me straight to the "Cloud Four, Inc" link. The screen reader navigation focus ring never gets lost. 🙂 

<img width="1137" alt="Screen Shot 2023-08-29 at 10 37 10 AM" src="https://github.com/cloudfour/cloudfour.com-patterns/assets/459757/3ed87b39-086e-49d4-932d-e849d95b18df">

Reply:
Maybe the H2 could use `hidden` instead of needing both `u-hidden-visually` and `aria-hidden`?

Reply:
Great suggestion!